app/src/androidTest/java/com/github/h3lp3rs/h3lp/StorageTest.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.h3lp3rs.h3lp
+
+import androidx.test.core.app.ApplicationProvider
+import com.github.h3lp3rs.h3lp.database.Databases
+import com.github.h3lp3rs.h3lp.database.MockDatabase
+import com.github.h3lp3rs.h3lp.signin.SignInActivity.Companion.globalContext
+import com.github.h3lp3rs.h3lp.storage.LocalStorage
+import com.github.h3lp3rs.h3lp.storage.Storages
+import com.github.h3lp3rs.h3lp.storage.Storages.*
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.resetStorage
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.storageOf
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import kotlin.random.Random
+
+class StorageTest {
+
+    // Dummy class for complex types
+    private data class Foo(val a1: Int, val a2: String)
+    // Useful variables
+    private lateinit var stor: LocalStorage
+    private val TEST_KEY = "KEY"
+    private val TEST_SEED = Random(0)
+    private val BYTES_PER_CHAR = 2
+    private val DELTA = 1e-10

```

Comment:
A few of these were common with the Database tests, it might be valuable to put them into a separate utils object/class for testing databases / storages (since we are essentially testing the same functionality)